src/lib/components/PostCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import Typography from '@material-ui/core/Typography';
+import CardMedia from '@material-ui/core/CardMedia';
+import { Box, Chip, makeStyles } from '@material-ui/core';
+import { mockData } from '../constants/postCard-config';
+
+const useStyles = makeStyles(
```

Comment:
pls make another file for styles - `PostCard.styles.ts`

Reply:
done